proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -108,7 +108,6 @@ message GetModuleDocumentationResponse {
 // ModuleDocumentation provides the name of the module and associated documentation.
 message ModuleDocumentation {
   string name = 1;
-  repeated string packages = 2;
```

Comment:
I assume the frontend changes have not been made to remove the dependency on this yet -- I will stamp, but try not to merge until the frontend is ready, since if `core` gets updated, then it would break the frontend ^^"